src/backend/backend.h

```diff
@@ -51,54 +52,41 @@ namespace snmalloc
 #endif
     };
 
-    /**
-     * Global state for the backend allocator
-     *
-     * This contains the various global datastructures required to store
-     * meta-data for each chunk of memory, and to provide well aligned chunks
-     * of memory.
-     *
-     * This type is required by snmalloc to exist as part of the Backend.
-     */
-    class GlobalState
-    {
-      friend BackendAllocator;
+    SNMALLOC_REQUIRE_CONSTINIT
+    static inline AddressSpaceManager<PAL> address_space;
 
-      AddressSpaceManager<PAL> address_space;
+    SNMALLOC_REQUIRE_CONSTINIT
+    static inline FlatPagemap<MIN_CHUNK_BITS, MetaEntry, PAL, fixed_range>
+      pagemap;
 
-      FlatPagemap<MIN_CHUNK_BITS, MetaEntry, PAL, fixed_range> pagemap;
-
-    public:
-      template<bool fixed_range_ = fixed_range>
-      std::enable_if_t<!fixed_range_> init()
-      {
-        static_assert(
-          fixed_range_ == fixed_range, "Don't set SFINAE parameter!");
+  public:
+    template<bool fixed_range_ = fixed_range>
+    static std::enable_if_t<!fixed_range_> init()
+    {
+      static_assert(fixed_range_ == fixed_range, "Don't set SFINAE parameter!");
 
-        pagemap.init();
+      pagemap.init();
 
-        if constexpr (fixed_range)
-        {
-          abort();
-        }
+      if constexpr (fixed_range)
```

Comment:
I think the `enable_if_t` makes this redundant.  (My fault, sorry).

Reply:
I'll remove it.